store/views.py

```diff
@@ -77,6 +132,52 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    book_id = request.POST.get('bid')
+    print(book_id)
+    book = BookCopy.objects.get(id=book_id)
+    user = User.objects.get(username = request.user.username)
+    if(book!=0):
+        book.borrower = None
+        book.status = True
+        book.borrow_date = None
+        book.save()
+        response_data = {
+        'message': 'success',
+        }
+    else:
+        response_data = {
+        'message': 'failure',
+    }
+
+    return JsonResponse(response_data)
 
 
+@csrf_exempt
+@login_required
+def ratingsystem(request):
+    if request.method == 'POST':
+        rate = request.POST.get('rate')
+        bid = request.POST.get('bid')
+        book = Book.objects.get(id = bid)
+        user = User.objects.get(username = request.user.username)
+        rateuser = rating()
+        rateuser.user = user
+        rateuser.rate = rate
+        rateuser.book = book
```

Comment:
You've not put a backend validation on the rating, so the user can easily put invalid values of rating.